packages/components/inputs/async-select-input/src/async-select-input.tsx

```diff
@@ -485,7 +523,12 @@ const AsyncSelectInput = (props: TAsyncSelectInputProps) => {
           // @ts-ignore: passed to the react-select components via `selectProps`.
           isCondensed={props.isCondensed}
           iconLeft={props.iconLeft}
-          controlShouldRenderValue={props.controlShouldRenderValue}
+          controlShouldRenderValue={
+            props.appearance === 'filter'
+              ? false
+              : props.controlShouldRenderValue
+          }
+          appea
```

Comment:
some leftover

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2980/commits/66e84c2e6b4acd52c9a5c6df98b208bdcf190f1a
Thank you